### PR TITLE
Home UI

### PR DESCRIPTION
**Pull request by chicky-dev**
https://edu-redmine.framgia.vn/issues/3556

### CODE REVIEW

**Comment by chicky-dev** [comment]
ready

File: app/src/main/java/com/framgia/vhlee/themoviedb/ui/home/BottomFragment.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.framgia.vhlee.themoviedb.ui.home;
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/vhlee/themoviedb/ui/home/BottomFragment.java]: remove bl.
ctrl alt L va O de format code

File: app/src/main/java/com/framgia/vhlee/themoviedb/ui/home/TopFragment.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.framgia.vhlee.themoviedb.ui.home;
+
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/vhlee/themoviedb/ui/home/TopFragment.java]: đây nữa

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by chicky-dev** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**